doc/content.tex

```diff
@@ -1,677 +0,0 @@
-         seasonal indices is subtracted from each period.
-        \item Random component is calculated by subtracting trend and seasonal component from original time series
-            $\hat{E}_t = y_t - \hat{T}_t - \hat{S}_t$.
-    \end{enumerate}
-
-    %%%%%%%%
-    \section{Augmented Dickey\,--\,Fuller Test} \label{sec:adf}
-    Time series statistical tests are often used for testing if there is particular characteristics present in time
-    series. Unit root tests are used whether a time series is stationary. In this work Augmented \\Dickey\,--\,Fuller
-    (ADF) test was chosen for unit root testing.
-
-    Its null hypothesis is \emph{$H_0$: time series contains a unit root}\,--\,it is not stationary. Outcome of this
-    test
-    is a negative ADF statistics. The more negative it is the stronger the rejection of the hypothesis.
-    The full form of ADF test is listen in \ref{eq:adf-test}.
-
-    \begin{gather} \label{eq:adf-test}
-        \Delta y_t = \alpha + \beta t + \gamma \Delta y_{t-1} + \dots + \delta{p-1} \Delta y_{t-p+1} + \epsilon_t \\
-        ADF = \frac{\hat{\gamma}}{SE(\hat{\gamma})} \label{eq:adf-stat}
-    \end{gather}
-
-    There are multiple variants of ADF test. Some of them leave out parts of equation \ref{eq:adf-test}. The most
-    important ones and widely used are:
-
-    \begin{itemize}
-        \item \emph{nc}\,--\,for regression with no constant nor time trend ($\beta t$)
-        \item \emph{c}\,--\,for regression with an intercept but no time trend ($\beta t$)
-        \item \emph{ct}\,--\,for regression with an intercept and time trend
-    \end{itemize}
-
-    Each of them is good for testing particular type of stationarity. For example for testing if there is trend
-    present in the time series \emph{c} version is the best choice.
-
-    The implementation of this test fits multiple linear regression model from equation \ref{eq:adf-test} and
-    calculates ADF statistics listed in \ref{eq:adf-stat}. SE denotes standard error of estimated $\hat{\gamma}$.
-
-    %%%%%%%%
-    \section{Seasonality Detection} \label{sec:period-detection}
-    Forecasting engine in Hawkular system does not have any inside information about period of a time series being
-    modelled. Therefore automatic period identification has to be implemented. In practice it is a difficult task and
-    result often differs from correct period, specially if there is significant noise present in the series
-    \cite{period-meteo}.
-
-    There are several approaches how to implement automatic period identification. The most used ones are based on
-    autocorrelation function (ACF) or spectral density \cite{period-hydman}. This work applies ACF method.
-
-    In the following Chart \ref{img:period-acf} ACF function of sine function is shown. The period of this
-    function is seven. There are patterns repeated every seven observations and it is decreasing to zero.
```

Comment:
uz predtym som toto zvazovala: decreasing by som nahradila "decreased" - kvoli tomu, ze je to proste fakt